src/main/scala/za/co/absa/spark_partition_sizing/DataFramePartitioner.scala

```diff
@@ -56,26 +56,39 @@ object DataFramePartitioner {
 
     def repartitionByPlanSize(minPartitionSize: Option[ByteSize], maxPartitionSize: Option[ByteSize]): DataFrame = {
 
-      def computeBlockCount(totalByteSize: BigInt, desiredSize: Long, addRemainder: Boolean): Int = {
+      def computeBlockCount(totalByteSize: BigInt, desiredSize: BigInt, addRemainder: Boolean): Int = {
```

Comment:
What is the reason for this change? (just wondering, it does not seem to make much of a difference, but maybe I am missing something)

Reply:
`sizeInBytes` is a BigInt and although in Enceladus it worked when passing a BigInt as a Long param, here it seems to fail compilation